src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -625,6 +602,38 @@ case class JesBackend(actorSystem: ActorSystem)
     }
   }
 
+  private def preemtped(errorCode: Int, backendCall: BackendCall) = errorCode == 13 && backendCall.preemptible
+
+  private def handleFailure(backendCall: BackendCall, errorCode: Int, errorMessage: Option[String], events: Seq[ExecutionEventEntry]) = {
+    import cromwell.util.IntUtil._
+
+    val taskName = s"${backendCall.workflowDescriptor.id}:${backendCall.call.unqualifiedName}"
+    val attempt = backendCall.key.attempt
+
+    if (errorMessage contains "Operation canceled at") {
+      AbortedExecutionHandle.future
+    } else if(preemtped(errorCode, backendCall)) {
+      val preemptedMsg = s"Task $taskName was preempted for the ${attempt.toOrdinal} time."
+      val max = backendCall.maxPreemption
+
+      if (backendCall.key.attempt < max) {
+        val e = new PreemptedException(
```

Comment:
Similar comment to what I made above. Are these exceptions merely vessels to carry error messages around the system or is the actual `PreemptedException` being used somewhere?